Assets/Scripts/Framework/Movement.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections;
+using FrameWork.Extensions;
+using UnityEngine;
+namespace Framework
+{
+    public abstract class Movement : MonoBehaviour
+    {
+        [Range(1,10)] private float _movementSpeed; 
+
+        private bool _moving;
+        public float MovementSpeed
```

Comment:
```suggestion
        private bool _isMoving;
        
        public float movementSpeed
```
1. bool kan beter met een werkwoord beginnen.
2. public varible is camelCasing.